src/libarena/lib.rs

```diff
@@ -393,9 +392,7 @@ struct TypedArenaChunk<T> {
     next: *mut TypedArenaChunk<T>,
 
     /// The number of elements that this chunk can hold.
-    capacity: usize,
-
-    // Objects follow here, suitably aligned.
+    capacity: usize, // Objects follow here, suitably aligned.
```

Comment:
This probably needs to have a `#[rustfmt_skip]`, I don't think we'll ever cope with this